src/plugins/presentation_util/public/components/field_picker/field_picker.scss

```diff
@@ -6,9 +6,7 @@
 .fieldPickerSelectable {
   height: $euiSizeXXL * 9; // 40 * 9 = 360px
 
-  &.fieldPickerSelectableLoading {
-    .euiSelectableMessage {
-      height: 100%;
-    }
+  .euiSelectableMessage {
```

Comment:
Very quick styling fix for when there are no matching fields that I figured I'd throw in here since it's so small:

**Before:**

![image](https://github.com/elastic/kibana/assets/8698078/6c18ce92-8224-4ba9-ac3f-93f266161dba)


**After:**

![image](https://github.com/elastic/kibana/assets/8698078/7f0441e3-62d7-4b0c-bc15-f96ff864e6ef)